package.json

```diff
@@ -106,5 +106,9 @@
     "ts-node": "~8.3.0",
     "tslint": "~6.1.0",
     "typescript": "~4.4.0"
+  },
+  "volta": {
```

Comment:
this makes sure I'm running the correct versions of node and npm when I work on this. If you're not using this, it's pretty nice https://volta.sh